Memmy-App/memmy
# Move Traverse tab into a Drawer

### PR Creator Checklist

Ensure you've checked the following before submitting your PR:


### Summary

- Moved the items in the Traverse tab to a drawer that can be accessed by swiping from the left side of the feed or a button on the top left of the feed.
- Remove the Traverse tab from the bottom nav bar.
- Added the Settings tab to the bottom nav bar.

### Screenshots

![Feed View](https://i.imgur.com/Fy1upXw.png)
![Traverse Drawer](https://i.imgur.com/0D0kHXn.png)

### Test Plan

1. Traverse Drawer
    1. Swipe from the left side of the screen to open the Traverse drawer.
    2. Swipe from the right side of the screen to close the Traverse drawer.
    3. Click the button in the Top Left corner of the feed to open the Traverse drawer.
    4. Interact with the items in Traverse drawer to verify it still functions.
    5. Click on the remaining portion of the Feed to close the Traverse drawer.
2. Navigation Bar
    1. Click on each tab to verify they are correct.
    2. Click on the Settings tab and its sub-screens to verify they load properly.
